src/libsyntax/feature_gate.rs

```diff
@@ -816,7 +816,7 @@ pub const BUILTIN_ATTRIBUTES: &'static [(&'static str, AttributeType, AttributeG
 ];
 
 // cfg(...)'s that are feature gated
-const GATED_CFGS: &'static [(&'static str, &'static str, fn(&Features) -> bool)] = &[
+const GATED_CFGS: &[(&str, &str, fn(&Features) -> bool)] = &[
```

Comment:
Yeah, clippy warned about the complex type, but I just figured I make use of my RFC 1623 (default static lifetime in statics).